libraries/c_sdk/standard/mqtt/src/iot_mqtt_context_connection.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * FreeRTOS MQTT V2.1.1
```

Comment:
This is going to be the Shim. I think it is good to start the versioning for shim with 1.0.0